packages/memtrace/memtrace.0.2.2/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Streaming client for Memprof"
+description: "Generates compact traces of a program's memory use."
+maintainer: ["opensource@janestreet.com"]
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+license: "MIT"
+homepage: "https://github.com/janestreet/memtrace"
+bug-reports: "https://github.com/janestreet/memtrace/issues"
+depends: [
+  "dune" {>= "2.3"}
+  "ocaml" {>= "4.11.0"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
Then we could do this:
```suggestion
    "@runtest" {with-test & ocaml:version < "5"}
```
But we should still add the post install warning message for base-domains compilers

Reply:
What's the issue with leaving this as-is, with the tests failing on OCaml 5? Gc.Memprof is currently broken in OCaml 5, so these tests failing seems like the correct outcome.